suite-common/wallet-utils/src/__tests__/accountUtils.test.ts

```diff
@@ -337,4 +340,113 @@ describe('account utils', () => {
         account.addresses = enhanceAddresses(accountInfo, account);
         expect(account.addresses.change).toEqual([]);
     });
+
+    it('isDebugOnlyAccountType', () => {
+        expect(isDebugOnlyAccountType('legacy', 'btc')).toBe(false);
+        expect(isDebugOnlyAccountType('segwit', 'btc')).toBe(false);
+        expect(isDebugOnlyAccountType('coinjoin', 'btc')).toBe(false);
+        expect(isDebugOnlyAccountType('taproot', 'btc')).toBe(false);
```

Comment:
please add regtest here

Reply:
added, tho it seems like it's not debug-only

Reply:
because it is debug only network, not debug account type